src/translations/en.json

```diff
@@ -1218,7 +1218,7 @@
           "title": "{count} {count, plural,\n  one {update}\n  other {updates}\n}",
           "unable_to_fetch": "Unable to load updates",
           "version_available": "Version {version_available} is available",
-          "more_updates": "+{count} updates",
+          "more_updates": "Show all updates",
```

Comment:
We should to make this consistent with what we do with `Updates`. Can't find the code what we are doing there.

Reply:
> I guess this will probably break translations...

It will not break, it will break if the translation requires an variable, but it is not provided